lib/org/openpsa/sales/handler/view.php

```diff
@@ -103,6 +105,20 @@ private function _populate_toolbar()
         $this->bind_view_to_object($this->_salesproject);
     }
 
+    private function is_pdf_creatable()
+    {
+        if ($this->_salesproject->state != org_openpsa_sales_salesproject_dba::STATE_LOST) {
+            $qb = org_openpsa_sales_salesproject_deliverable_dba::new_query_builder();
+            $qb->add_constraint('salesproject', '=', $this->_salesproject->id);
+            $qb->add_constraint('up', '=', 0);
+            $qb->add_constraint('state', '<', org_openpsa_sales_salesproject_deliverable_dba::STATE_DECLINED);
+            $qb->add_order('state');
+            $qb->add_order('metadata.created', 'DESC');
```

Comment:
since we're only counting results, the ordering is not important and can be removed